crates/ruma-common/src/push/action.rs

```diff
@@ -30,6 +30,13 @@ pub enum Action {
     _Custom(CustomAction),
 }
 
+impl Action {
+    /// Whether this action is an `Action::SetTweak(Tweak::Highlight(true))`.
+    pub fn is_highlight(&self) -> bool {
```

Comment:
Do we also want `is_*` methods for the other variants (of `Action`, not `Tweak`)?

Reply:
Afaik, currently we mostly use `Notify`, `Tweak::Highlight` and `Tweak::Sound`. `DontNotify` is the same as checking if `Notify` is absent.

Reply:
And by that I mean these are probably the only functions that would be useful.